app-backend/project/Version.scala

```diff
@@ -12,25 +12,25 @@ object Version {
   val awsStsSdk = "1.11.232"
   val betterFiles = "3.4.0"
   val caffeine = "2.3.5"
-  val cats = "1.0.1"
-  val catsEffect = "0.10.1"
-  val catsMeow = "0.1.3"
+  val cats = "1.4.0"
+  val catsEffect = "1.0.0"
+  val catsMeow = "0.2.0"
   val chill = "0.9.2"
   val circe = "0.9.1"
   val commonsIO = "2.5"
   val dnsJava = "2.1.8"
-  val doobie = "0.5.3"
+  val doobie = "0.6.0"
   val dropbox = "3.0.9"
   val elasticacheClient = "1.1.1"
   val ficus = "1.4.0"
   val findbugAnnotations = "3.0.1u2"
   val gatling = "2.2.4"
   val geotools = "17.1"
-  val geotrellis = "2.0.0"
-  val geotrellisServer = "0.0.1"
+  val geotrellis = "2.1.0-SNAPSHOT"
```

Comment:
Version `2.1.0` is now released

Reply:
Cool. This should be good to merge I think after updating that. Tests are passing now at least